docker/jsontest/JsonTests.pl

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env perl
```

Comment:
Tento script som nasiel na nete, a robi to iste ako nefunkcna sluzba http://time.jsontest.com/